networking_oneview/ml2/drivers/oneview/synchronization.py

```diff
@@ -55,6 +55,7 @@ def get_session(self):
 
     @common.oneview_reauth
     def synchronize(self):
+        common.check_valid_resources()
```

Comment:
In this case, the reason is not a wrong mapping anymore... but someone has deleted resources at OneView... should we consider a different/additional error message for users?

Reply:
@ricardoas I have updated the error messages, do you think we still need to have a better message for the synchronize part?

Reply:
I was thinking more on the bigpicture of failing on "driver init" or failing on "sync mechanism"... I'll think a bit more!